documents/privacy.md

```diff
@@ -249,14 +249,14 @@ Relating this to our privacy goals articulated above
     
 -   There is, unfortunately, a vulnerability for venues where diagnosed users have been - particularly in the case where an attacker has in mind a small area and a narrow time window, which they wish to evaluate for points of concern.
 
-However, while such attacks are possible, it nevertheless will take significant technical skill and effort to conduct such an attack, and the rewards are fairly minimal. Our expectation is that this will lead to few such attacks will be undertaken, and that this will be a negligible channel for disclosure of information about which venues were attended by people later diagnoses with COVID.
+However, while such attacks are possible, it nevertheless will take significant technical skill and effort to conduct such an attack, and the rewards are fairly minimal. Our expectation is that this will lead to few such attacks being undertaken, and that this will be a negligible channel for disclosure of information about which venues were attended by people later diagnosed with COVID.
```

Comment:
the attacker is targeting particular "venues" in this attack, if I understand correctly? ergo, "x location is a known contact site". but then, I agree, so what? as we move to exposure notification for increased movement, lots of locations will be "known contacts", which in and of itself, mitigations the attack value of this one. I want to make a joke like, "the attacker would have to dislike a venue a great deal to go to this kind of trouble". Might happen, but I don't think it will really be taken seriously by others, so even if it does happen once or twice, I'm guessing that only the perpetrator is going to care. No change to text. Just my 2 cts agreeing with the overall risk analysis.